src/main/java/newsonthego/NewsOnTheGo.java

```diff
@@ -23,6 +25,12 @@ public class NewsOnTheGo {
      */
     public static void main(String[] args) {
 
+        String url = "https://www.firstpost.com/tech/" +
```

Comment:
For now do you want to shift this to the ArticleScrapper file instead of the main function? Since the scrapper is not fully implemented yet.